reference/features/search_parameters.md

```diff
@@ -238,6 +238,10 @@ Configures which attributes will be retrieved in the returned documents.
 
 If no value is specified, `attributesToRetrieve` uses the [`displayedAttributes` list](/reference/features/settings.md#displayed-attributes), which by default contains all attributes found in the documents.
 
+::: note
+If an attribute has been removed from `displayedAttributes`, `attributesToRetrieve` will silently ignore it and the field will not appear in your returned documents.
```

Comment:
I prefer this, but if you and Gui prefer the current text then let's go with that 😄 

```suggestion
Attributes must be present in `displayedAttributes` to be retrievable with `attributesToRetrieve`.
```

Reply:
I think the current one is better as it also tells the user what happens if you don’t have your attribute in `displayedAttributes` 👀

Reply:
I'm fine with either. Since this is @maryamsulemani97's PR, I defer to her judgement.